tutorials/2 Front End User/2.3 My first QDesign/Algorithmic_Chip_Design.ipynb

```diff
@@ -199,7 +199,44 @@
    "metadata": {},
    "source": [
     "We can quickly check if we have designed the qubits well with an LOM analysis. \n",
-    "First we render the qubit to Ansys Q3D, presuming we have already opened Ansys."
+    "First we render the qubit to Ansys Q3D. Comment the \"LOM_q3d.open_ansys()\" command if you already have an Ansys session open."
```

Comment:
At the top of the notebook, please add a cell in "Markdown".

Suggestion:
# Prerequisite
You need to have a working local installation of Ansys